qemu/tests/balloon_hotplug.py

```diff
@@ -79,15 +77,10 @@ def enable_balloon_service():
 
         error_context.context("Hotplug balloon device for %d times" % (i+1),
                               logging.info)
-        out = new_dev.hotplug(vm.monitor)
-        if out:
-            err += "\nHotplug monitor output: %s" % out
-        # Pause
-        time.sleep(pause)
```

Comment:
I see there has one commit  c2fbee0560f7d9252c017e7226f7532eedc36797 to add `wait_for` for  `simple_unplug ` in avocado-vt, but not for `simple_hotplug`.
No need to wait for device be hot-pluged ?

Reply:
Yes, you are right, this is the correct logic, same with unplug. and my initial concern is why remove the `sleep` here without any change of simple_hotplug.

Reply:
Yes, you are right, thx

Reply:
Better to make `simple_hotplug` use the same logic with `simple_unplug` in avocado-vt to avoid potential problem if we update avocado-vt.

Reply:
@yanan-fu That's dangerous. We can accept unplug delay does not means we can accept hotplug delay, which might be a product bug. Before we have enough provement to show we can tolerant the hotplug delay, better keep it there, thus avoid possible bug leak.

Reply:
@vivianQizhu  Why it is might be a product bz?  I don't agree with this it.
As for the old design,  it do "sleep" between hotplug and verify_hotplug.  If we need keep it, we need add  wait_for in simple_hotplug too.

Reply:
@vivianQizhu  It is ok for me to keep simple_hotplug as no bug for it by now, that why i think it is `better to` not `should`.
And simple_hotplug is a common function for not only one kind device,  be part of not only one feature. I think it has nothing to do with update specific test case.
Also, still reserve my opinion, thanks !